### DISCUSSION TITLE
Intersecting 2 planes slows down the frame rate

### DISCUSSION THREAD

**Discussion by jairsnow** [enable3d/enable3d#158] [Help / Questions]
I'm not sure is a bug, maybe I've interpreting/doing something wrong.

Using `enable3d@0.22.0-dev.2` and three `125.2`

If I create 2 planes with a medium amount of segments (200*200) and I make them intersect the update rate drops to 1 frame per second.

```
const plane = this.physics.add.plane( {x: 0, y: 0 , z: 0, widthSegments: 200, heightSegments: 200, width: 20, height: 20, mass: 2, collisionFlags: 1}, {lambert: {color: 'yellow'}});
const plane2 = this.physics.add.plane( {x: 0, y: 0 , z: 0, widthSegments: 200, heightSegments: 200, width: 20, height: 20, mass: 2, collisionFlags: 1}, {lambert: {color: 'red'}});
plane2.rotateX(1.57);
plane2.body.needUpdate = true;
```

If I do the same thing with a box shape I don't observe any frame rate drop and anything continues to run smoothly

```
const box = this.physics.add.box({widthSegments: 200, heightSegments: 200, width: 20, height: 20, mass: 2, collisionFlags: 1}, {lambert: {color: 'yellow'}});
const box2 = this.physics.add.box({widthSegments: 200, heightSegments: 200, width: 20, height: 20, mass: 2, collisionFlags: 1}, {lambert: {color: 'red'}});
box2.rotateX(1.57);
box2.body.needUpdate = true;
```

Thanks for any explanation or help!

**Comment by yandeu** [comment]
Ammo.js does not have a Plane Body. If you add a plane, it will create a complex mesh body. 

Box on the other hand is a primitive, built-in shape.

This is why it is always faster to use a Box.

---

 _Btw, bullet (ammo.js) has a [Heightfield Shape](https://pybullet.org/Bullet/BulletFull/classbtHeightfieldTerrainShape.html), but I have never used it and I don't know how well it performs._
